nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/test/groovy/org/apache/nifi/web/StandardNiFiServiceFacadeSpec.groovy

```diff
@@ -49,6 +53,7 @@ class StandardNiFiServiceFacadeSpec extends Specification {
         SecurityContextHolder.getContext().setAuthentication(null);
     }
 
```

Comment:
The full suite of tests were previously ignored, however I had a test that could work.